dev-cli/src/commands/init.js

```diff
@@ -19,10 +19,10 @@ export async function init ({ lang = 'lua' }, name) {
 
 export const command = new Command()
   .description('Create an ao Process Source Project')
-  .usage('-l cpp <my-project-name>')
+  .usage('-l c <my-project-name>')
   .option(
     '-l, --lang <language:string>',
-    'The starter to use. Defaults to Lua. Options are "lua" and "cpp"'
+    'The starter to use. Defaults to Lua. Options are "lua" and "c"'
```

Comment:
This should not be in this PR. If anything, it should be a separate issue and PR, so that our team can look at it, by itself.